specs/moonbase-alphanet-specs-template.json

```diff
@@ -18,13 +18,10 @@
         "code": "<runtime_code>"
       },
       "palletBalances": {
-        "balances": [
-          ["15ozwjkTsjscc8raJFMdoEkzxE8NeqWWXwi4JGeC3DzAbGoi", 115292150460684697600],
-          ["16f4zjGB1XyXfdr7swiJFr91mSnHrb4CtEt95pz3htF8fHGQ", 115292150460684697600]
-        ]
+        "balances": []
```

Comment:
Are you intentionally removing these prefunded accounts?

Reply:
I moved them into new account (under EVM).